gtdynamics/utils/Trajectory.h

```diff
@@ -67,15 +82,17 @@ class Trajectory {
    */
   std::vector<PointOnLinks> phaseContactPoints() const {
     std::vector<PointOnLinks> phase_cps;
-    const auto &phases = walk_cycle_.phases();
-    for (size_t i = 0; i < repeat_; i++) {
-      for (auto &&phase : phases) {
-        phase_cps.push_back(phase.contactPoints());
-      }
+    for (auto &&phase : phases_) {
```

Comment:
what's difference wqith contact_points_? Document (I forget!)

Reply:
contact_points_ is a member vector of Trajectory with unique contact points of the trajectory.
`PhaseContactPoints` returns all contact points from all phases and may have repetitions.
I documented this.